dist/scss/variables.scss

```diff
@@ -111,6 +111,6 @@ $navigationHoverColor: #191D22;
 // Overlay
 // -------------------------
 $overlayContainerWidthSmall: 580px;
-$overlayContainerWidthMedium: 700px;
+$overlayContainerWidthMedium: 860px;
```

Comment:
The decision of the design section was to make the medium width larger.
That's right, this has an impact on already existing overlays.